iOS/rainbow/Controller/Camera/CameraController.swift

```diff
@@ -42,7 +42,7 @@ class CameraController: LuminaViewController {
         self.setTorchButton(visible: true)
         self.setCancelButton(visible: false)
         self.setSwitchButton(visible: false)
-        LuminaViewController.loggingLevel = .info
+        LuminaViewController.loggingLevel = .verbose
```

Comment:
Let's switch logging back to `.info` when we're running normally. We should only use other levels of logging when we debug an issue 😉